zoombot/zoombot/handler.go

```diff
@@ -117,10 +118,21 @@ func (h *Handler) zoomHandlerInner(msg chat1.MsgSummary) error {
 		h.ChatEcho(msg.ConvID, meeting.JoinURL)
 	case ZoomAPIError:
 		if err.Code == http.StatusTooManyRequests {
-			h.ChatEcho(msg.ConvID, err.Error())
-			return nil
+			if attempts > 5 {
+				h.ChatEcho(msg.ConvID, err.Error())
+				return nil
+			}
+			go func() {
```

Comment:
Maybe add a slight delay before retrying? It seems like if we are being ratelimited, we need to wait a little bit.

Reply:
hah the was the whole point of the routine 🤦‍♂